dime-research-standards/pillar-1-research-ethics/research-ethics-resources/informed-consent-templates/README.md

```diff
@@ -47,6 +49,10 @@ The higher the risk, the more extensive the informed consent procedures must be,
 - Signed consent form 
 - Consent specific to the recording medium (e.g., videotape)
 
+## Surveying minors
+When surveying minors (aged 8 to 17), an assent needs to be obtained from a parent or guardian unless an exemption is provided by the IRB. This [template](https://couhes.mit.edu/sites/default/files/documents/Assent-Form-2019-01-22.doc) can be used to develop an assent form. 
```

Comment:
May want to note that DPO approval would also have to be if not planning to get consent from guardian for any respondents below the age of consent. Privacy policy implementation guidelines also note that "minors under 18 or individuals under legal guardianship should be given the opportunity to assent to the use of their data whenever they are capable of doing so. The assent process should use language and processes suitable to the age and understanding of the respondent."

Reply:
added